README.md

```
@@ -0,0 +1,140 @@
+- **색상만 변경** 가능으로 나만의 컬렉션 구성
+- 다음 달 캘린더는 **1일 기준 7일 전**부터 미리 스크랩 가능
+
+### 랭킹 산정
+- **일별 랭킹**: 전날(어제) 태스크 완료율 기준
+- **월별 랭킹**: 해당 월 스크랩 수 기준
+
+## 📝 API 문서
+
+API 문서는 Swagger를 통해 제공됩니다.
+
+https://api.everyvent.cloud:8080/swagger-ui.html
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)